classes/03class/exercises/c03-docker06/mrcsmonteiro/README.md

```diff
@@ -0,0 +1,32 @@
+# C03-Docker06
+
+## Docker 
+- [Dockerfile](Dockerfile)
+
+## Command Execution Output
+- Commands used to build  the image:
+```
+$ docker build -t hello-name:v1 .
+Sending build context to Docker daemon  3.072kB
+Step 1/2 : FROM alpine:3.12.0
+3.12.0: Pulling from library/alpine
+df20fa9351a1: Pull complete
+Digest: sha256:185518070891758909c9f839cf4ca393ee977ac378609f700f60a771a2dfe321
+Status: Downloaded newer image for alpine:3.12.0
+ ---> a24bb4013296
+Step 2/2 : CMD echo "Hello ${NAME}"
+ ---> Running in 0ba16e441add
+Removing intermediate container 0ba16e441add
+ ---> 8aba9eaa09bb
+Successfully built 8aba9eaa09bb
+Successfully tagged hello-name:v1
+```
+
+- Commands used to execute the image and the output of container execution:
+```
+$ docker run -e NAME=Marcos hello-name:v1
```

Comment:
you should not be passing an environment variable...
you should execute docker `run image Marcos` only

Reply:
Thanks for the feedback, @kikobr82 . I have removed the ENV parameter from the docker run command and the corresponding variable from the Dockerfile.